### ISSUE TITLE
model fails when eradication is successful

### ISSUE THREAD

**Issue by dagaydos21** [ncsu-landscape-dynamics/pops_model_api#2]
If you manage to eradicate the pathogen, the model gives an error "run failed". It will create a run collection button for the scenario, probably because the first couple years of the scenario are valid. But it won't let you load/view the scenario after eradication.

**Comment by ChrisJones687** [comment]
@dagaydos21 I have deployed the fix for this and tested it to make sure it works but have only ran it once. If it works for well for you let me know and I will close the issue.

**Comment by dagaydos21** [comment]
I just tested it a couple of times, and it runs now. There's a couple of other small suggestions I have though that will make it clearer for the user.
1) The infections from 2018 are still visible, so it's not immediately obvious that the pathogen is eradicated. We should remove those like we have removed other managed infections.
2) It would be nice to have some sort of pop up message that lets the user know that there are no new infections. Something like "all infections have been removed"
